### DISCUSSION TITLE
北邮毕设模板交流

### DISCUSSION THREAD

**Discussion by way-zer** [BYRIO/BUPTBachelorThesis#16] [Show and tell]
仓库地址： https://github.com/way-zer/BUPTBachelorThesis
基于`tectonic`编译，环境搭建方便。已测试编译没问题。
同时将`main.tex`根据内容拆分为了若干`tex`文件，条例更清晰。

---

PS: 本科2023届，正在找模板。这里是BYRIO, 估计也有不少兄弟再找模板，留些链接

目前发现的：(欢迎补充)
* `Typst`的模板(看起来很新, 找到时还是16小时前更新)： https://github.com/QQKdeGit/bupt-typst/tree/master
* 研究生`Word`模板: https://bbs.byr.cn/#!article/Paper/45998
* 另一个`Latex`模板(貌似有点老): https://github.com/rioxwang/BUPTGraduateThesis/tree/master

**Comment by dasing-qing** [comment]
通知说论文查重系统只支持doc\docx

**Reply by guanidine** [reply]
Adobe Acrobat转word，除了公式乱码外，其他格式基本上都保持的不错
Pandoc的tex转docx，公式都能转出来，格式不对
两个可以结合起来用

**Reply by note286** [reply]
Microsoft Word可以直接打开PDF文件，也可以转换。

**Comment by guanidine** [comment]
教务处发新通知了，有人对过格式到底有哪些变化吗
目前只发现诚信声明跑到封面后面了，目录的参考文献和外文翻译好像要……，然后也不需要空格了。。。

**Comment by h-y1heng** [comment]
插图遇到问题：在章节中插入一张图，正好位于换页处，但是编译后这张图单独占了一页位于中间，而不是置顶再接下文内容，这要怎么办捏？

**Comment by Bang0518** [comment]
哥们，我是24届本科生，也在找模板。教务发的只有word排版要求，实际使用极其难受，想请问下你最后使用的模板是哪个repo的，感激不尽！

**Reply by Hyiker** [reply]
翻了一下今年的规范，对比23届似乎只更改了装订顺序：
> 毕业设计（论文）按下列顺序要求统一装订：封面→诚信声明、关于论文使用授权的说明(一页)→任务书→成绩评定表→中文摘要(含关键词)→外文摘要(含关键词)→目录→正文→参考文献→致谢→附录→外文资料及译文→开题报告→中期进展情况检查表→教师指导毕业设计(论文)记录表→提前进行毕业设计环节申请审批表(如有)→校外完成本科毕业设计（论文）审批表(如有)→变更指导教师申请表(如有)

别的可以直接沿用上一届的设置

**Comment by cindy-lee-bupt** [comment]
求问，如果想把表格内的内容居中该怎么办，在网上查到\centering但这个语句在\begin{bupttable}内并不识别，不能让表格居中

**Comment by liouhai** [comment]
期刊参考文献的［Ｊ］的位置和最新要求不一致哦

**Comment by cindy-lee-bupt** [comment]
您的信件已经收到，感谢。--This is an automatic reply, confirming that your e-mail was received.Thank you.
